inc/altis_config/class-qm-collector-altis-config.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Altis Config data collector for the Query Monitor plugin.
+ */
+
+namespace Altis\Dev_Tools\Altis_Config;
+
+use QM_Collector;
+
+class QM_Collector_Altis_Config extends QM_Collector {
+
+	public $id = 'altis-config';
+
+	function name() {
+		return _x( 'Altis Config', 'Menu item name for the Query Monitor plugin', 'altis' );
```

Comment:
```suggestion
		return esc_html_x( 'Altis Config', 'Menu item name for the Query Monitor plugin', 'altis' );
```

Reply:
If this is giving you a fatal error as you mentioned on Slack, @dasha, it's because this file is being loaded before QM loads.

Reply:
@johnbillion thanks, ended up reworking the approach. All works now :)

Reply:
@roborourke this is a `return` aren't we escaping on output?

Reply:
Presumably QM is, just going off one of our other QM panel implementations. Doesn't hurt to do this